src/cymbal.gleam

```diff
@@ -1,233 +1,30 @@
-import gleam/int
-import gleam/float
-import gleam/string
-
-/// A YAML document which can be converted into a string using the `encode`
-/// function.
-///
-pub opaque type Yaml {
-  Int(Int)
-  Bool(Bool)
-  Float(Float)
-  String(String)
-  Array(List(Yaml))
-  Block(List(#(String, Yaml)))
+import cymbal/decode.{parse_tokens, tokenize_lines}
+import cymbal/encode.{
+  type Yaml, Array, Block, Bool, Float, Int, Null, String, en, string,
```

Comment:
Put back everything that has been removed from this module please. There should be no breaking changes here.

Reply:
I felt like splitting the files would keep a more consistent api and this breaking change is possible right now as the downloads are so few but I see your point, will revert.

Reply:
A problem also arises where moving the encode.gleam file back into the cymbal.gleam file leads to a circular import. I believe a fix to this would be moving the `deocde` function from the `cymbal.gleam` file to the `decode.gleam` file but this would further splinter the api. I do not know the appropriate way to handle this and it will need some thought.

Reply:
To keep a clean api without breaking changes I believe combinding the code to a single file would be preffered. This can then be changed in a 2.0 but I feel like moving specifically the decoder to a `cymbal/decode.gleam` and keeping the encoder in the cymbal.gleam file would be unnececary.

Reply:
Any update on this?